### PR TITLE
Site added Saikaiscan

### LINKED ISSUE #1462: Please add Saikai Scan (pt-br)
The site: 'https://saikaiscan.com.br' is a scanlator that contains content from japanese, korean and mainly chinese works, both in comics and novels, in addition to even original brazilian novels, some contents that cannot be found in other online readers ..

### PR DESCRIPTION
**Pull request by newaccforhc**
Fixes #1462

### CODE REVIEW

File: src/web/mjs/connectors/Saikaiscan.mjs [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        super.label = 'Saikaiscan';
+        this.tags = [ 'manga', 'portuguese', 'webtoon' ];
+        this.url = 'https://saikaiscan.com.br';
+    }
+
+    async _getMangaFromURI(uri) {
+        let request = new Request(uri, this.requestOptions);
+        let data = await this.fetchDOM(request, 'div#project-content h2');
+        let id = uri.pathname;
+        let title = data[0].textContent;
```
Comment (XenonR) [inline src/web/mjs/connectors/Saikaiscan.mjs]: May add `.trim()`

File: src/web/mjs/connectors/Saikaiscan.mjs [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    }
+
+    async _getChapters(manga) {
+        let request = new Request(new URL(manga.id, this.url), this.requestOptions);
+        let data = await this.fetchDOM(request, 'div.chapters ul li a');
+        return data.map(element => {
+            return {
+                id: this.getRootRelativeOrAbsoluteLink(element, this.url),
+                title: element.textContent.trim(),
+                language: ''
```
Comment (XenonR) [inline src/web/mjs/connectors/Saikaiscan.mjs]: May remove unused parameter.

**Review by XenonR — APPROVED** [review]
Looking good.

File: src/web/mjs/connectors/Saikaiscan.mjs (hunk 2 above)
Comment (newaccforhc) [reply src/web/mjs/connectors/Saikaiscan.mjs]: haha... forgot to do that.. ^^'

File: src/web/mjs/connectors/Saikaiscan.mjs (hunk 1 above)
Comment (newaccforhc) [reply src/web/mjs/connectors/Saikaiscan.mjs]: Okay

**Pull request merged**